pkg/kubelet/pleg/evented.go

```diff
@@ -137,16 +139,15 @@ func (e *EventedPLEG) Stop() {
 		return
 	}
 	setEventedPLEGUsage(false)
-	close(e.stopCh)
-	close(e.stopCacheUpdateCh)
```

Comment:
Unrelated to this PR: Why is `Stop` exported? Isn't that only used internally?

As an exported API it's semantic would need to be better defined. For example, may it be called more than once? Not according to the implementation (closing a closed channel panics), but that's not obvious.